src/main/java/edu/wpi/grip/ui/pipeline/PipelineView.java

```diff
@@ -166,9 +166,11 @@ private SourceView findSourceView(Source source) {
     }
 
     /**
+     * This function is public because it is used by PreviewsView:onSocketPreviewChanged
+     * to order the list of previews.
      * @return The {@link StepView} that corresponds with the given step
      */
-    private StepView findStepView(Step step) {
+    public StepView findStepView(Step step) {
```

Comment:
@ThomasJClark Since this is being exposed publicly, should step have a `checkNotNull` on the param passed to it?